js/Worksheet.js

```diff
@@ -1155,8 +1156,9 @@ class Worksheet extends HTMLElement {
     toExcel(){
         const wb = XLSX.utils.book_new();
         const ws = XLSX.utils.aoa_to_sheet(
-            this.sheet.dataFrame.getDataArrayForFrame(
-                this.sheet.dataFrame
+            this.sheet.dataStore.getDataArray(
```

Comment:
a Frame's origin and corner are `Point` instances, but we decided that `getDataArray` should only accept coordinate pairs. So I don't think this part should be working

Reply:
yes see PR description -  all the excel and csv interfaces are messed up